src/transpiling/jsx_precompile.rs

```diff
@@ -1232,6 +1290,13 @@ impl JsxPrecompile {
       ))
     }
 
+    if let Some(espace_ident) = &self.import_jsx_escape {
```

Comment:
Maybe do `self.import_jsx_escape.take()` to avoid a clone? Not a big deal though.